libtenzir/builtins/aggregation-functions/all.cpp

```diff
@@ -63,14 +116,24 @@ class plugin : public virtual aggregation_function_plugin {
 
   [[nodiscard]] caf::expected<std::unique_ptr<aggregation_function>>
   make_aggregation_function(const type& input_type) const override {
-    if (caf::holds_alternative<bool_type>(input_type))
+    if (caf::holds_alternative<bool_type>(input_type)) {
       return std::make_unique<all_function>(input_type);
+    }
     return caf::make_error(ec::invalid_configuration,
                            fmt::format("all aggregation function does not "
                                        "support type {}",
                                        input_type));
   }
 
+  auto make_aggregation(invocation inv, session ctx) const
+    -> failure_or<std::unique_ptr<aggregation_instance>> override {
+    auto expr = ast::expression{};
+    TRY(argument_parser2::function("tql2.all")
```

Comment:
The name given to the argument parser shows up in user output.
~~~
error: expected additional positional argument `<expr>`
 --> <input>:1:11
  |
1 | summarize all()
  |           ^^^ 
  |
  = usage: tql2.all(<expr>)
  = docs: https://docs.tenzir.com/functions/tql2.all
~~~
Hence you just want to use `"all"`/`name()` here. Same goes for the other operators.